lib/classes/Variables.js

```diff
@@ -102,10 +102,31 @@ class Variables {
       const requiredConfigs = [
         _.assign({ name: 'region' }, provider.getRegionSourceValue()),
         _.assign({ name: 'stage' }, provider.getStageSourceValue()),
-        _.assign({ name: 'profile' }, {
+        {
```

Comment:
Does this `requiredConfigs` above mean that those are required to be present (e.g. users need to put in `region`, `stage`, `porfile`, etc.)? At least it reads like that. Pretty sure that's not the case. Just want to make sure that this is not a breaking change...

Reply:
yeah, the name is a little misleading. It works just fine when they're not set. They'r required to be prepopulated if they're set.